packages/calcite-components/src/utils/loadable.ts

```diff
@@ -111,3 +113,26 @@ export function setComponentLoaded(component: LoadableComponent): void {
 export function componentLoaded(component: LoadableComponent): Promise<void> {
   return promiseMap.get(component);
 }
+
+/**
+ * This helper util can be used to ensure a component is ready to be focused (The "componentDidLoad" stencil lifecycle method has been called and any internal elements are ready to be focused).
+ *
+ * Requires "setUpLoadableComponent" and "setComponentLoaded" to be called first.
```

Comment:
WDYT about simplifying this to ``requires `LoadableComponent` to be implemented`` (similar to [`T9nComponent`'s doc](https://github.com/Esri/calcite-components/blob/14036c1354419222bc5b839af8b68675b17ebf62/packages/calcite-components/src/utils/t9n.ts#L105)).